### PR TITLE
README :: Some style/grammar tweaks, and init/service script mention.…

### PR DESCRIPTION
**Pull request by yoosefi**
… Re: #1193

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,11 +54,19 @@ Fail2Ban should be correctly installed now. Just type:
 to see if everything is alright. You should always use fail2ban-client and
 never call fail2ban-server directly.
 
+Please note that the system init/service script is not automatically installed.
+To enable fail2ban as an automatic service, simply copy the script for your
+distro from the `files` directory to `/etc/init.d`. Example:
```
Comment (yarikoptic) [inline README.md]: `Example (on a Debian-based system):`

File: README.md [hunk 2]
```diff
@@ -6,13 +6,16 @@
...
-
-Fail2Ban is able to reduce the rate of incorrect authentications attempts
-however it cannot eliminate the risk that weak authentication presents.
+Fail2Ban scans log files like `/var/log/auth.log` and bans IP addresses
+having too many failed login attempts.
+It does this by updating system firewall rules to reject new connections
+from those IP addresses, for a configurable amount of time.
+Fail2Ban comes out-of-the-box ready to read many standard log files, such as those
+for sshd and Apache, and is easy to configure to read any log file you choose, for
+any error you choose.
```
Comment (yarikoptic) [inline README.md]: I would prefer "filled" paragraph to the width consistent with the rest (around 78?)

Comment (yoosefi) [inline README.md]: Sure.

**Comment by yarikoptic** [comment]
Thanks!

**Pull request merged**